virtual_ecosystem/models/soil/soil_model.py

```diff
@@ -234,20 +234,27 @@ def from_config(
             config, enzyme_classes=enzyme_classes
         )
 
+        # Load hydrology constants
+        hydro_constants = load_constants(config, "hydrology", "HydroConsts")
```

Comment:
This looks like a neat solution to the different constants, I should adopt this to my models...